src/app.tsx

```diff
@@ -1,61 +1,68 @@
-import { useState } from 'preact/hooks'
-import stave from './assets/stave.svg'
-import crotchet from './assets/crotchet.svg'
-import './app.css'
-import styled from 'styled-components'
-import { css } from 'styled-components'
+import { useState } from "preact/hooks";
+import stave from "./assets/stave.svg";
+import crotchet from "./assets/crotchet.svg";
+import "./app.css";
+import styled from "styled-components";
 
 export function App() {
-
   const Wrapper = styled.section`
-  padding: 4em;
-  background: papayawhip;
-`
+    padding: 4em;
+    background: papayawhip;
+  `;
 
   const Button = styled.button`
-  position: absolute;
-  top: 40em;
-  left: 36em;
-  font-size: 1em;
-  margin: 1em;
-  padding: 0.25em 1em;
-  border: 2px solid #BF4F74;
-  border-radius: 3px;
-`
+    position: absolute;
+    top: 40em;
+    left: 36em;
+    font-size: 1em;
+    margin: 1em;
+    padding: 0.25em 1em;
+    border: 2px solid #bf4f74;
+    border-radius: 3px;
+  `;
 
   const StyledStave = styled.img`
     position: relative;
     height: 50em;
     width: 50em;
-  `
+  `;
 
-  const StyledNote = styled.img.attrs(props => ({
-    $note: props.$note
-  })) <{ $note?: string; }>`
+  const StyledNote = styled.img.attrs((props) => ({
+    $note: props.$note,
+  }))<{ $note?: string }>`
     position: absolute;
     left: 35em;
-    top: ${props => props.$note};
+    top: ${(props) => props.note};
```

Comment:
This was the issue - there was simply an extraneous `$` prefixed to your `note` property